staff/tachi-fernandez/cv/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curiiculum Vitae</title>
+</head>
+<body>
+  <p>
+    <h1> Curriculum Vitae => Tachi Fernandez</h1>
+      <h2>ºDatos personales:</h2>
+        <section>
+           <li>-Name : Tachi Fernandez Luna</li>
+             <br></br>
+           <li>Adress : mi casa nº8</li> 
+             <br></br>
+           <li>D.N.I. : 12345678R</li>
+             <br></br>
+           <li>Nacionality : Puerto Rico </li>  
+        </section>
+             <br>========================================</br>
```

Comment:
<br> tag should be empty, cause its a break line. If you like to have a line you should use a <hr> tag

Reply:
thanks :+1: